Portfolio/src/assets/RouterLinkNav.scss

```diff
@@ -0,0 +1,45 @@
+$color-black: #1e1e1e;
```

Comment:
I guess this will be replaced by a variable in another branch

Reply:
I have a whole variables file in another branch, so this will be removed and instead I will import the variables file